types/use-deep-compare-effect/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for use-deep-compare-effect 1.2
+// Project: https://github.com/kentcdodds/use-deep-compare-effect#readme
+// Definitions by: JPeer264 <https://github.com/JPeer264>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { useEffect } from 'react';
+
+export const useDeepCompareNoEffect: typeof useEffect;
+
+export default useEffect;
```

Comment:
This will work, but for clarity you might want to declare a const of type `typeof useEffect` and export that. Currently it looks like you're exporting a value that's the same as `useEffect` rather than having the same type as `useEffect`.

Reply:
Alright, makes sense indeed. Updated